package.json

```diff
@@ -132,6 +132,37 @@
     "worker-loader": "2.0.0"
   },
   "resolutions": {
+    "@babel/core": "^7.2.2",
+    "@babel/generator": "^7.9.6",
+    "@babel/helper-annotate-as-pure": "^7.10.4",
+    "@babel/helper-create-class-features-plugin": "^7.10.4",
+    "@babel/helper-function-name": "^7.9.5",
+    "@babel/helper-get-function-arity": "^7.10.4",
+    "@babel/helper-member-expression-to-functions": "^7.10.4",
+    "@babel/helper-module-imports": "^7.10.4",
+    "@babel/helper-optimise-call-expression": "^7.10.4",
+    "@babel/helper-plugin-utils": "^7.10.4",
+    "@babel/helper-regex": "^7.10.4",
+    "@babel/helper-replace-supers": "^7.10.4",
+    "@babel/helper-split-export-declaration": "^7.10.4",
+    "@babel/helper-validator-identifier": "^7.10.4",
+    "@babel/highlight": "^7.10.4",
+    "@babel/parser": "^7.11.0",
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
+    "@babel/plugin-proposal-object-rest-spread": "^7.11.0",
+    "@babel/plugin-proposal-unicode-property-regex": "^7.10.4",
+    "@babel/plugin-syntax-class-properties": "^7.10.4",
+    "@babel/plugin-syntax-logical-assignment-operators": "^7.10.4",
+    "@babel/plugin-syntax-numeric-separator": "^7.10.4",
+    "@babel/plugin-syntax-object-rest-spread": "^7.8.3",
+    "@babel/plugin-syntax-typescript": "^7.10.4",
+    "@babel/plugin-transform-dotall-regex": "^7.10.4",
+    "@babel/plugin-transform-typescript": "^7.10.4",
+    "@babel/preset-env": "^7.11.5",
+    "@babel/preset-typescript": "^7.10.4",
+    "@babel/template": "^7.10.4",
+    "@babel/traverse": "^7.10.4",
+    "@babel/types": "^7.11.0",
```

Comment:
I added these to get an 8MB download gain on install, due to the 10MB initial bundle.

There may be some underlying issues with pushing some of these upgrades though.

That said, this points to some simple dependency management techniques as a good way to reduce install size...

Reply:
I've removed these for now, root cause at https://github.com/vercel/next.js/pull/12952 should be fixed separately and is an important fix to bring down bundle size by ~4MB.